docs/4.1. Behaviour - RTP Transport Type.md

```diff
@@ -87,9 +87,10 @@ SDP files contain both connection information and media information. When a para
 
 ## Interpretation of SDP Files
 
-All Senders and Receivers should comply with RFC 4566 (Session Description Protocol) when creating and parsing SDP files. Multicast Senders and Receivers should additionally comply with RFC 4570 (SDP Source Filters). Three examples are given below, showing an SDP file and the transport parameters that would be presented at the `/staged` endpoint by a Receiver that has parsed the file.
+All Senders and Receivers should comply with RFC 4566 (Session Description Protocol) when creating and parsing SDP files. Multicast Senders and Receivers should additionally comply with RFC 4570 (SDP Source Filters). Three examples are given below, showing an SDP file and the transport parameters that would be presented at the `/staged` endpoint by a single-legged Receiver that has parsed the file.
+A Receiver sets the `rtp_enabled` parameter to true for each leg it has configured from the SDP file.
```

Comment:
Is it worth adding an 'unless overridden by the requested `transport_params`'?

Reply:
Good idea, this OK?